packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -104,102 +136,209 @@ class MouseTracker extends ChangeNotifier {
     _router.removeGlobalRoute(_handleEvent);
   }
 
-  // The pointer router that the mouse tracker listens to for events.
+  /// Find annotations at a given offset in global logical coordinate space
+  /// in visual order from front to back.
+  ///
+  /// [MouseTracker] uses this callback to know which annotations are affected
+  /// by each device.
+  ///
+  /// The order between the returned annotations is important since it
```

Comment:
The order of the returned annotations....

This should also say what the order needs to be.